### PR TITLE
🔖 3.3.0 Release

### PR DESCRIPTION
**Pull request by kunwarsahni01**
* Added docs for ADI Expander
* Added Release Notes

### CODE REVIEW

**Comment by nathan-moore** [comment]
There are some other changes that went in that require a doc change. The controller length amount needs some, mutex delete, a new overload on Controller, and various other things

File: v5/api/cpp/rtos.rst [hunk 1]
```diff
@@ -800,6 +800,40 @@ Analogous to `mutex_create <../c/rtos.html#mutex-create>`_.
 Methods
 -------
 
+delete
```
Comment (nathan-moore) [inline v5/api/cpp/rtos.rst:803]: This doesn't exist. It's the destructor instead.

**Review by nathan-moore — APPROVED** [review]
Otherwise LGTM

**Review by HotelCalifornia — APPROVED** [review]
let's go

**Pull request merged**